x/ethbridge/client/cli/tx.go

```diff
@@ -100,7 +101,7 @@ func GetCmdCreateEthBridgeClaim(cdc *codec.Codec) *cobra.Command {
 }
 
 // GetCmdBurn is the CLI command for burning some of your eth and triggering an event
-//nolint:lll
+//nolint
```

Comment:
For some reason the linter complains about code it this file being the same, but it IS different.